README.md

```diff
@@ -4,6 +4,17 @@
 
 A maven spring boot cart application.
 
+## Configurations
+
+Environment variables can be provided to configure the cart service.
+
+**Keycloak configurations**
+* CATALOG_ENDPOINT - Endpoint of the catalog endpoint
```

Comment:
`CATALOG_ENDPOINT - Endpoint of the catalog endpoint` this is not part of keycloak configuration

Reply:
Already move out of the Keycloak configurations part.